tests/php_loops_and_numbers/test.sh

```diff
@@ -0,0 +1,192 @@
+#!/bin/bash
+
+NONE='\033[00m'
+BOLD='\x1b[1m'
+ITALIC='\x1b[3m'
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+
+total_tests=6 # only 6 tests are currently being done, this value should be changed if tests increases
+
+echo -n "Enter First Name : "
+read first_name
+
+echo -n "Enter Last Name : "
+read last_name
+
+set_folder() {
+    first_name=$(echo "$first_name" | tr '[:upper:]' '[:lower:]')
+    first_letter=$(echo "${last_name:0:1}" | tr '[:lower:]' '[:upper:]')
+    last_substring=$(echo "${last_name:1}" | tr '[:upper:]' '[:lower:]')
+
+    folder_name="$first_name$first_letter$last_substring"
+}
+
+set_folder
+
+submission_directory="$PWD/../../submissions"
+path_to_folder="$submission_directory/$folder_name"
+path_to_php_folder="$path_to_folder/php"
+path_to_log="$submission_directory/../tests/logfile.json"
+
+check_php_folder_existence() {
+    folder_count=$(ls $path_to_folder | grep "php" -c)
+    if [ $folder_count -eq 0 ]
+    then
+        php_folder_exists=0
+        echo ""
+        echo -e "${BOLD}Checking the database folder ...${NONE}"
+        echo -e "${RED}Your folder should be named ${BOLD}\"php\"${NONE} ${RED}and located in the folder with your First and Last name. (${BOLD}\"$folder_name/database\"${NONE}${RED})${NONE}"
```

Comment:
@Ireti203 The folder you are checking is wrong. I think I mentioned something around this in the last PR. Please help work on that. Everything else looks good though, well done.

Reply:
Fixed. @ToluAkin